src/lib/curl/base.h

```diff
@@ -32,6 +32,16 @@ extern "C" {
 #include <curl/curl.h>
 #include <freeradius-devel/server/request.h>
 #include <freeradius-devel/util/event.h>
+#include <freeradius-devel/server/module.h>
+
+#define FR_CURL_SET_OPTION(_x, _y)\
+do {\
+	if ((ret = curl_easy_setopt(randle->candle, _x, _y)) != CURLE_OK) {\
+		char const *_option;\
```

Comment:
Looks like you removed the debug statement, which is why CIT is throwing an error and saying _option isn't used.

The original log message was:

`REDEBUG("Failed setting curl option %s: %s (%i)", option, curl_easy_strerror(ret), ret);`